.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
Try running this https://stackoverflow.com/questions/1143796/remove-a-file-from-a-git-repository-without-deleting-it-from-the-local-filesyste so you can remove from Git, all the files, which were added before you updated .gitignore.

Reply:
Also, we need to delete and ignore conduit-connect-airtabl.exe.